src/behavior-considered-undefined.md

```diff
@@ -72,11 +72,14 @@ Please read the [Rustonomicon] before writing unsafe code.
   * An integer (`i*`/`u*`), floating point value (`f*`), or raw pointer obtained
     from [uninitialized memory][undef], or uninitialized memory in a `str`.
   * A reference or `Box<T>` that is [dangling], misaligned, or points to an invalid value.
-  * Invalid metadata in a wide reference, `Box<T>`, or raw pointer:
+  * Invalid metadata in a wide reference, `Box<T>`, or raw pointer. The requirement
+    for the metadata is determined by the type of the unsized tail:
     * `dyn Trait` metadata is invalid if it is not a pointer to a vtable for
-      `Trait` that matches the actual dynamic trait the pointer or reference points to.
+      `Trait` that matches the actual dynamic type the pointer or reference points to.
```

Comment:
Preexisting but how can this be a requirement for wide _pointers_, whose data portion is allowed to dangle, I assume?

Reply:
Hm... yeah the actually implemented requirement is that the `trait` matches the trait given in the pointer type, i.e. a `*const dyn Debug` that points to a vtable for `Display` is UB.